src/Scene/DefaultUpdater.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
> I think we need a different updater for each one of these strategies.

`const process` allows to manage the different structures.
You can instance a new `Updater` with different process for each data structure. The difference is the `processList`, it must be adapt to the structure.

And I return to the static and generic `processNode` (like SSE and RTC) to be given to the `Updater`.